quelpa-use-package.el

```diff
@@ -37,17 +37,35 @@
 
 ;;; Code:
 
-(defvar quelpa-use-package-inhibit-loading-quelpa nil
+(defgroup quelpa-use-package nil
+  "Quelpa handler for `use-package'."
+  :prefix "quelpa-use-package"
+  :group 'use-package)
+
+(defcustom quelpa-use-package-inhibit-loading-quelpa nil
   "If non-nil, `quelpa-use-package' will do its best to avoid
 loading `quelpa' unless necessary. This improves performance, but
-can prevent packages from being updated automatically.")
+can prevent packages from being updated automatically."
+  :type 'boolean
+  :group 'quelpa-use-package)
+
+(defcustom quelpa-use-package-as-source-of-truth nil
+  "If non-nil, only packages installed via this will be managed by `quelpa'.
```

Comment:
It may be unclear what "this" refers to.

As well, it seems unclear what "managed by Quelpa" means.